core/services/relay/evm/bindings.go

```diff
@@ -3,48 +3,55 @@ package evm
 import (
 	"fmt"
 
-	"github.com/ethereum/go-ethereum/common"
-
-	"github.com/smartcontractkit/chainlink-common/pkg/types"
+	commontypes "github.com/smartcontractkit/chainlink-common/pkg/types"
 )
 
-type Bindings map[string]methodBindings
+type bindings map[string]map[string]binding
```

Comment:
nit, why not call `map[string]binding ` readBindings for readability, also add a comment that string is contract name